src/pages/Login/index.js

```diff
@@ -22,9 +30,9 @@ function Login() {
       email: profile.getEmail(),
       idToken,
     };
-
     // let's send the data to our backend.
     dispatch(userActions.onUserSignin(googleData));
+    updateQueryString(getQueryVariable("referrer"));
```

Comment:
There are two issues here -
1. You are doing the state update outside of the `useEffect` hook, this will cause a re-render each time.
2. Why aren't we using the single call we make in line 34, for sending the referrer name ? 
It calls - https://github.com/navgurukul/bhanwari-devi/blob/da55469ff742b93fab7f36eab908aa292fbe7536/src/components/User/redux/api.js#L12
Which in turn, makes a `POST` request to `${process.env.REACT_APP_MERAKI_URL}/users/auth/google`
Why can't we handle it in the same call itself ?